Kelvin/nba_challenge.py

```diff
@@ -0,0 +1,74 @@
+
+# TODO: Create variable here for number of 3 pt shots made by Fred VanVleet
+# TODO: Create variable here for number of 3 pt shots made by James Harden
+print("Challenge 2.1:")
+
+jamal_murray_3pts_made = 46 
+fred_van_vleet_3pts_made = 43 
+james_harden_3pts_made = 37
+print(jamal_murray_3pts_made)
+print(fred_van_vleet_3pts_made)
+print(james_harden_3pts_made )
+
+print("Challenge 2.2:")
+print(f"In the 2020 NBA playoffs, Jamal Murray made {jamal_murray_3pts_made} 3 point shots")
+# TODO: Create print statement here for Fred VanVleet
+print(f"In the 2020 NBA playoffs, Fred VanVleet made {fred_van_vleet_3pts_made} 3 point shots")
+# TODO: Create print statement here for James Harden
+print(f"In the 2020 NBA playoffs, James Harden made {james_harden_3pts_made} 3 point shots")
+
+
+print("Challenge 2.3: Store the number of three point shot attempts in variables for each player")
+# TODO: Create variable here for number of 3 pt shot attempts by Jamal Murray
+jamal_murray = '93 shot attempts'
+# TODO: Create variable here for number of 3 pt shot attempts by Fred VanVleet
+fred_van_vleet = '110 shot attempts'
+# TODO: Create variable here for number of 3 pt shot attempts by James Harden
+james_harden = '109 shot attempts'
+print(jamal_murray)
+print(fred_van_vleet)
+print(james_harden)
+
+print("Challenge 2.4: Build on your print statement")
+# TODO: Copy the three print statements you wrote in Challenge 2.2 and extend them to also print
+# the number of three point shots for each player. E.g., output should be similar to
+# "In the 2020 NBA playoffs, player X made Y 3 point shots and Z 3 point shot attempts."
+
+print(f"In the 2020 NBA playoffs, Jamal Murray made {jamal_murray_3pts_made}, 3 point shots {jamal_murray}")
+print(f"In the 2020 NBA playoffs, Jamal Murray made {fred_van_vleet_3pts_made}, 3 point shots {fred_van_vleet}")
+print(f"In the 2020 NBA playoffs, Jamal Murray made {james_harden_3pts_made }, 3 point shots {james_harden}")
+
+print("Challenge 2.5: Calculate, store, and print the three point percentage for each player")
+# TODO: Calculate the three point percentage, which is given by `three points made/three point attempts`
+
+jamal_murray_percentage = 46 / 93
+print(jamal_murray_percentage)
+fred_van_vleet_percentage = 45 / 110
+print(fred_van_vleet_percentage)
+james_harden_perfectage = 46 / 109
+print(james_harden_perfectage)
```

Comment:
They were hoping that the variables be used: for example: `jamal_murray_3pts_made/jamal_murray_3pts_attempts`

Where above, you would set `jamal_murray_3pts_attempts = 93`

Reply:
for each player